### ISSUE TITLE
Install setup for DBB

### ISSUE THREAD

**Issue by gkeuken** [mikefultonbluemix/zbrew#33]
Setup scripting/config files for DBB installation

**Comment by MikeFultonDev** [comment]
@gkeuken ... I think you are still working on this but noticed that the prereq of java 8 and z/OS 2.1 isn't done yet... 
Also have to figure out how to incorporate git into here

**Comment by gkeuken** [comment]
Yes .. I left this issue open to revisit pre-req  since we were changing how pre-req's were done and I didn't want to rework anything.

Customers might not use SMP/E to install Java .... so we might have to take a different approach to verify it's version (like checking java -version).

**Comment by MikeFultonDev** [comment]
@gkeuken how about we require they use SMP/E for our pre-req check but we annotate the pre-req to indicate it can be bypassed because it can be non-SMP/E installed (in which case we spit out a warning?) We would need that for git too (has the same option to be manually installed)

**Comment by gkeuken** [comment]
yes ..I agree ... at this point we should stick to verifying pre-req checks via SMP/E only ..  there is the restart function that can also be used to bypass pre-req check =>  zbrew -r smpconfig install ${sw} 
(smpconfig is the next logical step are pre-req check)
